store/src/main/java/com/dropbox/android/external/store4/SourceOfTruth.kt

```diff
@@ -92,7 +92,7 @@ interface SourceOfTruth<Key, Input, Output> {
          * @param delete function for deleting records in the source of truth for the given key
          * @param deleteAll function for deleting all records in the source of truth
          */
-        fun <Key : Any, Input : Any, Output : Any> fromNonFlow(
+        fun <Key : Any, Input : Any, Output : Any> from(
```

Comment:
After reading through your diff and thinking more, I think that your original approach was right - lets use the names to point people to the most common defaults and the the RTFM if they need something different. In this case I'm landing on the side that the confusion of having name overload with different parameter names outweighs the consistency benefits.

Reply:
to make sure i understand correctly, you mean keep SoT methods as `from` for both cases? (like in this CL?)

Reply:
no. I've come around to you're original approach of naming the SoT methods `from` and `fromNonFlow` Overload separation by named properties seems very subtle. It is what you would do in Swift, but there the names are mandatory, not optional.